### PR TITLE
Submitting Italian translations

### PR DESCRIPTION
**Pull request by zovanni**

### CODE REVIEW

File: src/messages.coffee [hunk 1]
```diff
@@ -355,6 +391,7 @@ define ->
         pt_br: -> "Você ainda não tem nenhum grid! Crie os seu próprios…"
         es_es: -> "¡No hay ninguna retícula definida! Guarda la tuya o…"
         pl_pl: -> "Nie masz jeszcze wzorców! Zapisz swój własny lub…"
+        it_it: -> "Non hai ancora alcuna griglia. Salvane una o..."
```
Comment (cameronmcefee) [inline src/messages.coffee]: Please replace the `...` with `…` <- you can copy the character here.

**Comment by cameronmcefee** [comment]
:heart: 🇮🇹 

Thanks, this looks good to me. Bonus points for even finding this repo.

**Comment by cameronmcefee** [comment]
Whoops, just noticed this didn't get merged. I'll get this into the next version. Sorry about that.

**Comment by zovanni** [comment]
hehehe no problem,
thanks, all the best



Giovanni Di Maggio
*www.giodimaggio.com <http://www.giodimaggio.com/>*
*it.linkedin.com/in/giovannidimaggio1/
<http://it.linkedin.com/in/giovannidimaggio1/>*

2016-07-29 22:19 GMT+02:00 Cameron McEfee <notifications@github.com>:

**Pull request merged**